content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -76,7 +76,7 @@ the hostname of the pod. For example, given a Pod with `hostname` set to
 The Pod spec also has an optional `subdomain` field which can be used to specify
 its subdomain. For example, a Pod with `hostname` set to "`foo`", and `subdomain`
 set to "`bar`", in namespace "`my-namespace`", will have the fully qualified
-domain name (FQDN) "`foo.bar.my-namespace.svc.cluster-domain.example`".
+domain name (FQDN) "`foo.bar.my-namespace.pod.cluster-domain.example`".
```

Comment:
AFAIK, there is no such domain like `xxx.pod.cluster-domain.example`. The pod FQDN is just under the `.svc.cluster-domain.example` domain and it is by design. 
So I am afraid no change is need. :)

Reply:
Good to know :), thanks for your reply! 

I'll close the PR.